src/components/new-members/index.js

```diff
@@ -1,17 +1,45 @@
 import React from 'react';
 import { membersContext } from '@store/members/members-context';
 import Card from '@components/member-card';
+import { useRouter } from 'next/router';
+import { userContext } from '@store/user/user-context';
+import Link from 'next/link';
 
 import styles from '@components/new-members/new-members.module.scss';
 
 // returns card which shows details of new member
-const renderNewMember = (newMember) => (
+const RenderNewMemberCard = (newMember, optionKey) => (
   <div className={styles.containerForNewMember}>
-    <Card developerInfo={newMember} isMember={false} />
+    <Card optionKey={optionKey} developerInfo={newMember} isMember={false} />
   </div>
 );
 
-const NewMemberList = () => {
+const RenderNewMember = (newMember, optionKey) => {
+  const { username, first_name, last_name } = newMember;
+  const { query } = useRouter() || { query: { dev: false } };
+  const { dev } = query;
+  const { isSuperUserMode } = userContext();
+  return isSuperUserMode && dev && optionKey ? (
+    <Link
+      prefetch={false}
+      href={{
+        pathname: '/[id]',
+        query: {
+          first_name: `${newMember ? first_name : ''}`,
+          last_name: `${newMember ? last_name : ''}`,
+        },
+      }}
+      as={`/${username}`}
+      key={username}
+    >
+      {RenderNewMemberCard(newMember, optionKey)}
+    </Link>
+  ) : (
+    RenderNewMemberCard(newMember, optionKey)
+  );
+};
```

Comment:
Are we creating a profiles page for new members? 🤔  If yes then it should be part of different PR and not this.
cc - @ankushdharkar